core/geometry/src/polyface/IndexedPolyfaceWalker.ts

```diff
@@ -0,0 +1,350 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/** @packageDocumentation
+ * @module Polyface
+ */
+
+import { IndexedEdgeMatcher, SortableEdgeCluster } from "./IndexedEdgeMatcher";
+import { IndexedPolyfaceVisitor } from "./IndexedPolyfaceVisitor";
+import { IndexedPolyface } from "./Polyface";
+
```

Comment:
I though navigation around facets, across edges, and around vertices are already available.

Reply:
Before this PR, `PolyfaceData` only stored enough data to walk around a facet: the face loop of each facet is stored in the `pointIndex` array.